lib/Runtime/Executor/ThreadPoolExecutor.cpp

```diff
@@ -373,7 +416,8 @@ void ThreadPoolExecutor::handleDeviceManagerResult(
     if ((node->children).empty()) {
       // If the node has no children, propagate its outputs to the result
       // PlaceholderBindings for the run.
-      propagateOutputPlaceholders(executionState, std::move(ctx));
+      propagateOutputPlaceholders(executionState,
```

Comment:
Can we trace this and/or `propagatePlaceholdersForNode`? I'm curious if all this tensor cloning is expensive.

Reply:
@SplitInfinity We are tracing handleResult as a block (see the TRACE_EVENT_BEGIN above), do you mean it would be useful to have an additional breakdown for propagateOutputPlaceholders?

Reply:
Yeah, that's what I mean.

Reply:
Cool I'll add it.

Reply:
More so `propagatePlaceholdersForNode` because it calls `Tensor::clone` a lot.

Reply:
I've only really examined resnet50 on CPU devices, but copying doesn't seem like a significant cost right now.